src/Tests/InstrEngineTests/TestAppRunner/Program.cs

```diff
@@ -60,7 +60,26 @@ static int Main(string[] args)
 
                 return entryPointDelegate(entryPointArgs);
             }
-            
+        }
+
+        private static Assembly LoadAssemblyFromDisk(string assemblyPath)
+        {
+            FileStream assemblyStream;
+            BinaryReader assemblyReader;
+
+            assemblyStream = new FileStream(assemblyPath, FileMode.Open, FileAccess.Read);
+            using (assemblyReader = new BinaryReader(assemblyStream))
+            {
+                byte[] assemblyData = assemblyReader.ReadBytes((int)assemblyStream.Length);
+                return AppDomain.CurrentDomain.Load(assemblyData);
+            }
+        }
+
+        private static Assembly OnAssemblyResolve(object sender, ResolveEventArgs args)
```

Comment:
>OnAssemblyResolve [](http://example.com/codeflow?start=32&length=17)

Please add some comments about why we do this.

Reply:
This is OK, but some explanation of *why* the dependent assemblies aren't getting loaded would be good.